python/src/ExponentialFactory_doc.i.in

```diff
@@ -4,16 +4,30 @@
 Available constructor:
     ExponentialFactory()
 
-The parameters are estimated by method of moments:
+Let :math:`\bar{x}` be the minimum mean. 
+Let :math:`x_{(1)}` be the minimum sample value.
+The parameter :math:`\hat{\gamma}` is estimated by method of maximum 
```

Comment:
This formulation is strange: it gives the impression that \hat{\gamma} needs to be estimated when it is actually the estimator (as shown by the equation). The parameter that needs to be estimated is \gamma.

Reply:
You are absolutely right. Fixed.